tests/test_charts.py

```diff
@@ -290,33 +310,37 @@ def test_axis(chart_2d):
     axis.tick_labels = "4e"
     axis.tick_locations = tlocs_large  # Add some more variety to labels
     chart_2d.show()
+
     assert tuple(axis.tick_labels) == tuple(to_vtk_scientific(f"{loc:.4e}") for loc in tlocs_large)
     assert vtk_array_to_tuple(axis.GetTickLabels()) == tuple(
         to_vtk_scientific(f"{loc:.4e}") for loc in tlocs_large
     )
     assert axis.GetNotation() == charts.Axis.SCIENTIFIC_NOTATION
     assert axis.GetPrecision() == 4
-    axis.tick_locations = None
-    axis.tick_labels = None
-    chart_2d.show()
-    assert np.allclose(axis.tick_locations, tlocs0)
-    assert np.allclose(axis.GetTickPositions(), tlocs0)
-    assert tuple(axis.tick_labels) == tuple(tlabels0)
-    assert vtk_array_to_tuple(axis.GetTickLabels()) == tuple(tlabels0)
```

Comment:
Could you explain what makes this removed?

Reply:
This will have to be resolved in #2868, but fundamentally I disagree with this test and I think that it only indicates that the chart plotting was already broken within VTK. Here, resetting the tick locations should set them to their defaults, but instead it resets it to their prior set locations. In VTK 9.1 this works (probably a bug), and in VTK 9.2 it resets them to their defaults (desired behavior).

So, in short, this portion of the test was removed because it is testing for invalid VTK 9.1 behavior.

Reply:
I got it. Thanks.